atc/db/container_repository.go

```diff
@@ -362,3 +363,45 @@ func (repository *containerRepository) DestroyFailedContainers() (int, error) {
 
 	return int(failedContainersLen), nil
 }
+
+func (repository *containerRepository) DestroyUnknownContainers(workerName string, reportedHandles []string) (int, error) {
+	dbHandles, err := repository.queryContainerHandles(sq.Eq{
+		"worker_name": workerName,
+	})
+	if err != nil {
+		return 0, err
+	}
+
+	unknownHandles := diff(reportedHandles, dbHandles)
```

Comment:
should we stop here? as there are no `unknownHandles` to move to `destroying` 🤔

Reply:
done